control/controlclient/direct.go

```diff
@@ -1213,6 +1288,13 @@ func (c *Direct) SetDNS(ctx context.Context, req *tailcfg.SetDNSRequest) (err er
 			metricSetDNSError.Add(1)
 		}
 	}()
+	// Try noise first.
+	// TODO(maisem/bradfitz): remove the fallback before 1.24.
```

Comment:
I'd rather do it now. We can turn off noise if we find problems from the /key?v=28 URL.

That is: if there's noise, use noise only.

Reply:
That would still require a client restart to drop the cached key, no?

Reply:
`tailcfg.Debug.Exit` 🤷‍♂️